### ISSUE TITLE
Fix label clicking

### ISSUE THREAD

**Issue by moromis** [GumusLab/clustergrammer#15]
Right now, if zoomed fully in, the top couple labels do not register via `singleClicking`, in this screenshot:
<img width="510" alt="image" src="https://user-images.githubusercontent.com/4188237/189517639-433a79df-d227-4206-b504-9fe7bcef628e.png">
The first label you can click on the y-axis is `DEF vs GHI`

**Comment by tschaka1904** [comment]
Is this related to the wrong clicking behaviour? I noticed this behaviour:

When I double-clicking on a label on the y-axis, it does do some sort of re-ordering. But the little grey indicator triangle becomes red two rows underneath.
<img width="1298" alt="Screenshot 2022-09-27 at 21 34 47" src="https://user-images.githubusercontent.com/5269116/192619624-dd1ee549-5b7d-4f8d-9f73-bfdcab031745.png">

**Comment by moromis** [comment]
@tschaka1904 Yeah, it is related. I think in general the conversion of mouse coordinates to webgl coordinates and also how it transforms with regards to the aspect ratio is wrong. I think it belies a larger underlying issue, as I think it's caused by inserting the legend component, the canvas width changing, and then never updating `viz_dim`.

**Comment by tschaka1904** [comment]
I can confirm the issue about the legend. Not showing it resolves this issue.
